src/Types.tsx

```diff
@@ -0,0 +1,17 @@
+export interface Todo {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number
+}
+
+export interface User {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+}
+
+export interface TodoWithUser extends Todo {
+  user: User | undefined;
```

Comment:
To show that `user` can be undefined you can use optional parameter symbol

```suggestion
  user?: User;
```